scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java

```diff
@@ -67,12 +67,12 @@ public OutputView(OutputController controller) {
 
         this.controller.getModel().addJobOutputListener(this);
 
-        this.noOutputMessage = "No output available<br><br>" +
-                               "Click <strong>Finished tasks output</strong> to retrieve logs for finished tasks<br>" +
-                               "Use <strong>Streaming Output</strong> to auto-fetch logs for running tasks.";
+        this.noOutputMessage = "No output available<br><br><br>" +
+        					   "Click <strong>Finished Tasks Output</strong> to retrieve logs for already finished tasks. For all the finished Tasks within the Job, or for the selected Task. The command does work when Job is still in Running state, as well as when Job is Finished. In case of large number of Tasks (over 20), because the Tasks are paginated, when Selected Job is activated, it will add to the display the logs of the current page of Tasks.<br><br>"+
```

Comment:
"For all the finished Tasks within the Job, or for the selected Task. ": @cdiop I think this sentence should be rephrased.